handlers/utils.go

```diff
@@ -140,6 +140,9 @@ func getBusiness(r *http.Request) (*business.Layer, error) {
 // and provides a default value if it's not present.
 func clusterNameFromQuery(queryParams url.Values) string {
 	cluster := queryParams.Get("clusterName")
+	if cluster == "" {
+		cluster = queryParams.Get("cluster")
```

Comment:
This sounds familiar. We renamed this parameter a while ago because some environments (like OpenShift) have cluster as a hardcoded param that collide.

@nrfox i think this might mean we forgot other places to rename the parameter.